### ISSUE TITLE
Usability for objects with known magnitudes

### ISSUE THREAD

**Issue by cylammarco** [blancmatter/SNRcalcs#9]
The current implementation is set up for survey mode. It is not the most straight forward to use the calculator for objects with know magnitudes, a.k.a. follow-up mode.

**Comment by blancmatter** [comment]
I've just had a think. What Iain was asking for was;

* A box where an objects magnitude can be entered
* A value of exposure time is given, based on the instrument and observing condition values

## Quick Implementation

This could be added below the instrument options, and would need to be an auto updating table which lists exposure time against the (in the case below) 5 seeing values. If the user selects the Plot mode to be SNR or Sky brightness, then the table would update to reflect these values.


![image](https://user-images.githubusercontent.com/45846630/71888997-26ad2600-3139-11ea-8e38-e0db2bd26c00.png)


## Best Implementation

We should really refactor the Calculation code (same calculation for spectroscopy and imaging, but with different mode calls in the function) and store the Instrument ReadNoise, Zeropoints, Filter Options, etc in a JSON structure. We can have a single point of modification of parameters, then build a couple of different visualisations of the ETC (i.e. the Graph based 'Survey  Mode', and a more table based Calculator) which are calling this singular piece of js. 

In short re-structure rather than add on!
